src/Appwrite/Utopia/Response/Model/Collection.php

```diff
@@ -14,38 +14,55 @@ public function __construct()
                 'type' => self::TYPE_STRING,
                 'description' => 'Collection ID.',
                 'default' => '',
-                'example' => '5e5ea5c16897e',
+                'example' => '',
             ])
-            ->addRule('$permissions', [
-                'type' => Response::MODEL_PERMISSIONS,
-                'description' => 'Collection permissions.',
-                'default' => new \stdClass,
-                'example' => new \stdClass,
-                'array' => false,
+            ->addRule('$read', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Collection read permissions.',
+                'default' => '',
+                'example' => '',
+                'array' => true
+            ])
+            ->addRule('$write', [
+                'type' => self::TYPE_STRING,
+                'description' => 'Collection write permissions.',
+                'default' => '',
+                'example' => '',
+                'array' => true
             ])
             ->addRule('name', [
                 'type' => self::TYPE_STRING,
                 'description' => 'Collection name.',
                 'default' => '',
-                'example' => 'Movies',
+                'example' => '',
+            ])
+            ->addRule('attributes', [
+                'type' => self::TYPE_STRING,
```

Comment:
Use the Attribute model constant as the type here. A good example for how nested objects should look is available in `Appwrite\Utopia\Response\Model\Project`